WordPress/gutenberg
# Page List: Added color support

## What?
Add color support to the Page List block.
Part of : https://github.com/WordPress/gutenberg/issues/43245

## Why?
Page List block is missing color support.

## How?
Adds the color support via block.json

## Testing Instructions

- Go to the Global Styles setting ( under appearance > editor > styles > edit styles > blocks ).
- Make sure that the Page List block's color is configurable via Global Styles.
- Verify that Global Styles are applied correctly in the editor and frontend.
- Edit the template/page, add Page List block and apply the color for  background and links.
- Verify that block color styles take precedence over global block colour settings.
- Verify that block color display correctly in both the editor and frontend.

## Screenshots or screencast 

Backend:
![pagelistcolorbackend](https://github.com/user-attachments/assets/2c6381ee-33d6-440c-bc93-21f00d3ce17d)

Frontend:
![pagelistcolorfrontend](https://github.com/user-attachments/assets/e7003a92-8ed7-4f21-9ca6-fc778ac7ebaf)
